src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,4 +1,14 @@
-// Add the required types and props
-export const UserInfo = () => (
-  <>UserInfo markup</>
+import React from 'react';
+import { User } from '../../types/User';
+
+type Props = {
+  user: User | null;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => (
+  <>
+    return (
+    {(user?.name && user?.email) && <a className="UserInfo" href={`mailto:${user.email}`}>{user.name}</a>}
+    );
+  </>
```

Comment:
You need check your task requirements 
This component should render only if we have user

Destructure user
```suggestion
  const { name, email } = user

  return (
    <a className="UserInfo" href={`mailto:${email}`}>{name}</a>
  );
```